data/db_demo.yaml

```diff
@@ -112,8 +112,8 @@ telescope:
     =id: P60
   - name: Nordic Optical Telescope
     nickname: NOT
-    lat: 28.75
-    lon: 17.88
+    lat: 28.7572
```

Comment:
Glad we caught these ;)

Reply:
Yes definitely ! Another strong point for the map i guess :)